processing/pcap_filter.py

```diff
@@ -61,6 +71,12 @@ class PcapFilter(Processing):
     """
     key = "pcap_filter"
 
+    def __init__(self, *args, **kwargs):
+        super(PcapFilter, self).__init__(*args, **kwargs)
+        self.host_ignore_list = deepcopy(HOST_IGNORE_LIST)
```

Comment:
you can use `set.copy()`
also add a documentation explaining why a set is needed here and not a list

Reply:
set.copy() is shallow copy